### DISCUSSION TITLE
pane-border-format missing "current" functionality?

### DISCUSSION THREAD

**Discussion by rodhash** [tmux/tmux#4083] [General]
Would be nice if we could make different style for active and inactive pane border, like we have for Window status (`window-status-current-format` and `window-status-format`)

This isn't available as of today, right? I don't see it in the docs.. or is there a way I can customize this?

I know we can customize the pane style (colors).. but can we customize the format?

Something like this: `pane-border-format` and `pane-border-current-format` _(this one doesn't exist today)_

Thanks

**Comment by MayeulDestouchesMF** [comment]
I think you can have the same functionality by using the `#{active_pane}` boolean combined with the ternary operator `?`. For instance, you could have:

`set -g pane-border-format "#{?#{active_pane}, #T (active), #T (inactive)}"`

Beware that you may need to escape commas with `#,` when using the ternary `?` operator:

`set -g pane-border-format "#{?#{active_pane}, [bg=red#,fg=blue] #T (active), [bg=blue#,fg=red] #T (inactive)}"`
